### DISCUSSION TITLE
Navigation Display not functioning

### DISCUSSION THREAD

**Discussion by Londstride50** [flybywiresim/aircraft#2826] [Help] [answered]
Ive tried uninstalling reinstalling versions 5 and 5.1 to solve a issue, but Im having no luck.

The ND screen is stuck on one type of display 'Rose', none of the buttons or knobs work on the the Navigation Display, ie Distance knob, Arc, Plan, Nav or any of the buttons WPT,ARPT,NDB...

I think the ND screen is basically not functioning at all.

![ND](https://user-images.githubusercontent.com/76822298/103421785-2140bb00-4b96-11eb-8e31-b92e6d173146.JPG)

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
Do you have any liveries from asobo/marketplace? As there are reports that those liveries are incompatible with the mod, also try clearing out your community folder except for the a32nx to check if any other mods are having a conflict with the a32nx

**Comment by Londstride50** [comment]
That's interesting, I did actually have the Asobo Test Pilot livery .

Ive removed it and also reinstalled the original Asobo A320 via the content manager and replaced the a32xnx folder.

Its fixed, thanks for the advice :-)
